### PR TITLE
[devtools-translate] Chinese - What's New in DevTools (Chrome 105)

### LINKED ISSUE #3440: [devtools-translate] Chinese - What's New in DevTools (Chrome 105)
Post: https://developer.chrome.com/blog/new-in-devtools-105/
Translated by: August 19

Translator: @aquaMAX
Reviewers: @liuliangsir, @louisyoong, @xyugroup, @hanselfmu

### PR DESCRIPTION
**Pull request by aquaMAX**
Fixes #3440

Changes proposed in this pull request:

- Updated index.md for WNDT 105

### CODE REVIEW

File: site/zh/blog/new-in-devtools-105/index.md [hunk 1]
```diff
@@ -29,169 +28,177 @@ draft: true
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:28]: item 5, please update the zh whats-new.md file as well.

File: site/zh/blog/new-in-devtools-105/index.md [hunk 2]
```diff
@@ -29,169 +28,177 @@ draft: true
...
+如果您现在重播用户流程，它将失败，因为 **录制面板** 在录制期间不会自动捕获鼠标悬停事件。 要解决此问题，请 [手动添加步骤](/docs/devtools/recorder/reference/#add-and-remove-steps) 在单击菜单项之前将鼠标悬停在选择器上。
 {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/GY1ZkqEU3zbGmhEKoblN.png", alt="Support mouse over event in the Recorder", width="800", height="488" %}
 
-Chromium issue: [1257499](https://crbug.com/1257499)
+Chromium 议题: [1257499](https://crbug.com/1257499)
 
 
 <!-- ## Largest Contentful Paint (LCP) in the Performance insights panel {: #lcp } -->
-
+## 性能洞察面板中的最大内容绘制 (LCP) {: #lcp }
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:59]:
```suggestion
## Performance insights 面板中的最大内容绘制 (LCP) {: #lcp }
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 3]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 <!-- LCP is an important, user-centric metric for measuring [perceived load speed](https://web.dev/user-centric-performance-metrics/#types-of-metrics). You can now find out the critical paths and root causes of a [Largest Contentful Paint (LCP)](https://web.dev/lcp/). -->
-
+最大内容绘制是一个重要的、以用户为中心的衡量指标，用于衡量 [感知加载速度](https://web.dev/user-centric-performance-metrics/#types-of-metrics)。 您现在可以找出 [最大内容绘制 (LCP)](https://web.dev/lcp/) 的关键路径和根本原因。
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:61]:
```suggestion
最大内容绘制 (LCP) 是测量[感知加载速度](https://web.dev/user-centric-performance-metrics/#types-of-metrics)的一个以用户为中心的重要指标。 您现在可以找出 [最大内容绘制 (LCP)](https://web.dev/lcp/) 的关键路径和根本原因。
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 4]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 <!-- In a [performance recording](/docs/devtools/performance-insights/#record), click on the LCP badge in the **Timeline**. In the **Details** pane, you can view the LCP score, learn how to fix resources that slow down the LCP and see the critical path for the LCP resource. -->
-
+在 [性能录制](/docs/devtools/performance-insights/#record) 中，单击 **时间线** 中的 LCP 徽章。 在 **细节** 窗格中，您可以查看 LCP 分数，了解如何修复降低 LCP 速度的资源并查看 LCP 资源的关键路径。
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:63]:
```suggestion
在 [性能录制](/docs/devtools/performance-insights/#record) 中，单击 **时间线** 中的 LCP 徽章。 在 **细节** 边栏中，您可以查看 LCP 分数，了解如何修复降低 LCP 速度的资源并查看 LCP 资源的关键路径。
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 5]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 <!-- See [Performance Insights](/docs/devtools/performance-insights/) to learn how to get actionable insights and improve your website’s performance with the panel. -->
-
+请参阅 [绩效洞察](/docs/devtools/performance-insights/)了解如何通过面板获得可操作的见解并提高网站的性能。
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:65]:
```suggestion
参阅 [Performance insights](/docs/devtools/performance-insights/) 文档以了解如何通过此面板获得可行性的建议并提高网站的性能。
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 6]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/NZZJ1FzXxqj2U2NR0U53.png", alt="LCP in the Performance insights panel", width="800", height="751" %}
 
-Chromium issue: [1326481](https://crbug.com/1326481)
+Chromium 议题: [1326481](https://crbug.com/1326481)
 
 
 <!-- ## Identify flashes of text (FOIT, FOUT) as potential root causes for layout shifts {: #foit-fout } -->
-
+## 将闪烁的文本（FOIT、FOUT）识别为布局变化的潜在根本原因 {: #foit-fout }
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:72]:
```suggestion
## 识别闪烁的文本（FOIT、FOUT）为布局偏移的潜在根本原因 {: #foit-fout }
```

Comment (jecfish) [reply site/zh/blog/new-in-devtools-105/index.md:72]: refer to https://web.dev/i18n/zh/cls/

File: site/zh/blog/new-in-devtools-105/index.md [hunk 7]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 <!-- The **Performance insights** panel now detects [flash of invisible text (FOIT) and flash of unstyled text (FOUT)](https://web.dev/preload-optional-fonts/#font-rendering) as potential root causes for layout shifts. -->
-
+**性能洞察**面板现在可以检测 [flash of invisible text (FOIT) 和 flash of unstyled text (FOUT)](https://web.dev/preload-optional-fonts/#font-rendering) 作为布局变化的根本原因。
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:74]:
```suggestion
**Performance insights**面板现在可以检测 [flash of invisible text (FOIT) 和 flash of unstyled text (FOUT)](https://web.dev/preload-optional-fonts/#font-rendering) 作为布局偏移的根本原因。
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 8]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 <!-- To view the potential root causes of a layout shift, click on a screenshot in the **Layout shifts** track. -->
-
+如要查看布局偏移的潜在根本原因，请单击 **Layout shifts** 轨道中的屏幕截图。
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:76]:
```suggestion
单击 **Layout shifts** 轨道中的屏幕截图，以便查看布局偏移的潜在根本原因。
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 9]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 <!-- See [Optimize WebFont loading and rendering](https://web.dev/optimize-webfont-loading/) to learn the technique to prevent layout shifts.  -->
-
+请参阅 [优化 WebFont 加载和渲染](https://web.dev/optimize-webfont-loading/) 以了解防止布局变化的技术。
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:78]:
```suggestion
请参阅 [优化 WebFont 加载和渲染](https://web.dev/optimize-webfont-loading/) 以了解防止布局偏移的技术。
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 10]
```diff
@@ -29,169 +28,177 @@ draft: true
...
-Chromium issues: [1334628](https://crbug.com/1334628), [1328873](https://crbug.com/1328873)
+Chromium 议题: [1334628](https://crbug.com/1334628), [1328873](https://crbug.com/1328873)
 
 
 <!-- ## Protocol handlers in the Manifest pane {: #manifest } -->
-
+## Manifest 窗格中的协议处理程序 {: #manifest }
 <!-- You can now use DevTools to test the [URL protocol handler registration](https://web.dev/url-protocol-handler/) for [Progressive Web Apps (PWA)](https://web.dev/learn/pwa/). -->
-
+您现在可以使用 DevTools 测试 [渐进式网络应用 (PWA)](https://web.dev/learn/pwa/) 的 [URL 协议处理程序注册](https://web.dev/url-protocol-handler/)。
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:87]:
```suggestion
您现在可以使用 DevTools 测试[渐进式网络应用 (PWA)](https://web.dev/learn/pwa/) 的 [URL 协议处理程序注册](https://web.dev/url-protocol-handler/)。
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 11]
```diff
@@ -29,169 +28,177 @@ draft: true
...
+修复了重复的规范 URL 的 source map URL 索引。 以前，由于重复的规范 URL ,在某些文件中没有激活断点。
 
-
-Chromium issue: [1335338](https://crbug.com/1335338), [1333411](https://crbug.com/1333411)
+Chromium 议题: [1335338](https://crbug.com/1335338), [1333411](https://crbug.com/1333411)
 
 
 <!-- ## Miscellaneous highlights {: #misc } -->
- 
+ ## 其他亮点 {: #misc }
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:191]:
```suggestion
 ## 其他的更新 {: #misc }
```

Comment (jecfish) [reply site/zh/blog/new-in-devtools-105/index.md:191]: Need to fix some previous translations as well.

File: site/zh/blog/new-in-devtools-105/index.md [hunk 12]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 <!-- These are some noteworthy fixes in this release: -->
-
+这里是一些本版本中需要注意的修复:
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:193]:
```suggestion
以下是此版本中一些值得注意的修复：
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 13]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 <!-- - Properly remove a local storage key value pair from the table in the **Application** > **Local Storage** pane when it is deleted. ([1339280](https://crbug.com/1339280)) -->
+- 删除时，从 **Application** > **Local Storage** 窗格中的表中正确删除本地存储键值对。 ([1339280](https://crbug.com/1339280))
 <!-- - The color previews are now correctly displayed when viewing CSS files in the **Sources** panel. Previously, their positions were misplaced. ([1340062](https://crbug.com/1340062)) -->
+- 在 **Sources** 面板中查看 CSS 文件时，现在可以正确显示颜色预览。 此前，他们的位置错位了。 ([1340062](https://crbug.com/1340062))
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:197]:
```suggestion
- 在源代码面板中查看 CSS 文件时，正确显示颜色预览。 此前，它们的位置错位了。 ([1340062](https://crbug.com/1340062))
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 14]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 
 
 <!-- ## Miscellaneous highlights {: #misc } -->
- 
+ ## 其他亮点 {: #misc }
 <!-- These are some noteworthy fixes in this release: -->
-
+这里是一些本版本中需要注意的修复:
 <!-- - Properly remove a local storage key value pair from the table in the **Application** > **Local Storage** pane when it is deleted. ([1339280](https://crbug.com/1339280)) -->
+- 删除时，从 **Application** > **Local Storage** 窗格中的表中正确删除本地存储键值对。 ([1339280](https://crbug.com/1339280))
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:195]:
```suggestion
- 从 **Application** > **Local Storage** 窗格中的表正确删除本地存储键值对。 ([1339280](https://crbug.com/1339280))
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 15]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 <!-- - The color previews are now correctly displayed when viewing CSS files in the **Sources** panel. Previously, their positions were misplaced. ([1340062](https://crbug.com/1340062)) -->
+- 在 **Sources** 面板中查看 CSS 文件时，现在可以正确显示颜色预览。 此前，他们的位置错位了。 ([1340062](https://crbug.com/1340062))
 <!-- - Consistently display the CSS flex and grid items in the **Layout** pane, as well as display them as badges in the **Elements** panel. Previously, the flex and grid items were randomly missing in both places. ([1340441](https://crbug.com/1340441), [1273992](https://crbug.com/1273992)) -->
+- 在 **Layout** 窗格中始终显示 CSS flex 和 grid 项，并在 **Elements** 面板中将它们显示为徽章。 以前，flex 和 grid 项目在这两个地方都随机丢失。 ([1340441](https://crbug.com/1340441), [1273992](https://crbug.com/1273992))
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:199]:
```suggestion
- 在 **Layout** 边栏中始终显示 CSS flex 和 grid 元素，并在 **元素** 面板中将它们显示为徽章。 以前，flex 和 grid 元素会在这两个地方都随机丢失。 ([1340441](https://crbug.com/1340441), [1273992](https://crbug.com/1273992))
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 16]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 <!-- - A new **Creator Ad Script** link is available for [ad frames](https://chromium.googlesource.com/chromium/src/+/master/docs/ad_tagging.md#adtracker) if DevTools found the script that caused the frame to be labeled as an ad. You can open a frame via **Application** > **Frames**. ([1217041](https://crbug.com/1217041)) -->
+- 新增 **Creator Ad Script** 链接可用于 [广告框架] (https://chromium.googlesource.com/chromium/src/+/master/docs/ad_tagging.md#adtracker)。 如果 DevTools 搜索到了导致框架被标记为广告的脚本。 您可以通过 **Application** > **Frames** 打开框架。 ([1217041](https://crbug.com/1217041))
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:201]:
```suggestion
- [广告框架] (https://chromium.googlesource.com/chromium/src/+/master/docs/ad_tagging.md#adtracker)里新增 **Creator Ad Script** 链接。 当 DevTools 搜索到了导致框架被标记为广告的脚本时，您可以通过 **Application** > **Frames** 打开框架。 ([1217041](https://crbug.com/1217041))
```

**Review by jecfish — CHANGES_REQUESTED** [review]
Kudos for your first translation, Zui Yong. I review the post partially. Will ask for other translators to help review as well

File: site/zh/blog/new-in-devtools-105/index.md (hunk 1 above)
Comment (aquaMAX) [reply site/zh/blog/new-in-devtools-105/index.md:28]: where can i find the whats-new.md file?

Comment (jecfish) [reply site/zh/blog/new-in-devtools-105/index.md:28]: You can find it with this path: https://github.com/GoogleChrome/developer.chrome.com/tree/main/site/_includes/partials/devtools/zh

**Comment by louisyoong** [comment]
Review done.

**Review by xyugroup — COMMENTED** [review]
Suggested 

-您现在可以录制面板中对用户流程进行逐步重播和设置断点
+您现在可以设置断点并在录制面板中对用户流程进行逐步重播。

**Review by xyugroup — COMMENTED** [review]
Suggested 

-您现在可以录制面板中对用户流程进行逐步重播和设置断点
+您现在可以设置断点并在录制面板中对用户流程进行逐步重播。

File: site/zh/blog/new-in-devtools-105/index.md [hunk 17]
```diff
@@ -29,169 +28,177 @@ draft: true
 
 
 <!-- ## Step-by-step replay in the Recorder {: #recorder } -->
-
+## 录制面板中的逐步重播 {: #recorder }
```
Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-105/index.md:31]:
```suggestion
## 录制面板中的分步重放 {: #recorder }
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 18]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 <!-- You can now set a breakpoint and replay a user flow step by step in the **Recorder** panel. -->
-
+您现在可以**录制**面板中对用户流程进行逐步重播和设置断点。
```
Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-105/index.md:33]:
```suggestion
您现在可以**录制**面板中对用户流程进行分步重放和设置断点。
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 19]
```diff
@@ -29,169 +28,177 @@ draft: true
...
 <!-- To set a breakpoint, click on the blue dot next to a step. Replay your user flow, the replay will pause before executing the step. From here, you can continue the replay, execute a step, or cancel the replay. -->
-
+点击步骤侧边的蓝点以设置断点，然后重播您的用户流程。重播将在执行该步骤前暂停。 从这里，您可以选择继续重播、执行一个步骤或取消重播。
```
Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-105/index.md:35]:
```suggestion
点击步骤侧边的蓝点以设置断点，然后重放您的用户流程。该重放动作将在执行该步骤前暂停。 从这里，您可以选择继续重放、执行一个步骤或取消重放。
```

**Comment by jecfish** [comment]
Thanks all for the reviews! @aquaMAX , the reviews are completed. Once you address the comments, I think we are good to go!

File: site/zh/blog/new-in-devtools-105/index.md [hunk 20]
```diff
@@ -29,169 +28,177 @@ draft: true
 
 
 <!-- ## Step-by-step replay in the Recorder {: #recorder } -->
-
+## 录制面板中的分步重放 {: #recorder }
 <!-- You can now set a breakpoint and replay a user flow step by step in the **Recorder** panel. -->
-
+您现在可以**录制**面板中对用户流程进行逐步重播和设置断点。
```
Comment (liuliangsir) [inline site/zh/blog/new-in-devtools-105/index.md:33]: It seems to be omitted.
```suggestion
您现在可以**录制**面板中对用户流程进行分步重放和设置断点。
```

File: site/zh/blog/new-in-devtools-105/index.md [hunk 21]
```diff
@@ -13,10 +13,9 @@ tags:
   - new-in-devtools
   - devtools
   - chrome-105
-draft: true
 ---
 
-*感谢 [Jim Lim @xyugroup](https://www.linkedin.com/in/jim-lim-539a5638) 提供的翻译*。
+*感谢 [Poong Zui Yong](https://www.linkedin.com/in/zui-yong-poong-1b507b14/) 提供的翻译*。
 
 {% include 'partials/devtools/zh/banner.md' %}
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-105/index.md:20]:
```suggestion
{% include 'partials/devtools/zh/banner.md' %}
{% YouTube id='bHw_56RiVsg' %}
```

File: site/_includes/partials/devtools/zh/whats-new.md [hunk 22]
```diff
@@ -3,16 +3,17 @@
...
-* [Attach Wasm debugging information at runtime](/zh/blog/new-in-devtools-105/#wasm)
-* [Support live edit during debugging](/zh/blog/new-in-devtools-105/#live-edit)
-* [View and edit @scope at rules in the Styles pane](/zh/blog/new-in-devtools-105/#scope)
-* [Sourcemap improvements](/zh/blog/new-in-devtools-105/#sourcemaps)
-* [Miscellaneous highlights](/zh/blog/new-in-devtools-105/#misc)
+* [录制面板中的分步重放](/zh/blog/new-in-devtools-105/#recorder)
+* [支持录制面板中的鼠标悬浮事件](/zh/blog/new-in-devtools-105/#recorder-hover)
+* [性能洞察面板中的最大内容绘制 （LCP）](/zh/blog/new-in-devtools-105/#lcp)
+* [将闪烁的文本（FOIT、FOUT）识别为布局变化的潜在根本原因](/zh/blog/new-in-devtools-105/#foit-fout)
+* [Manifest 窗格中的协议处理程序] (/zh/blog/new-in-devtools-105/#manifest)
```
Comment (jecfish) [inline site/_includes/partials/devtools/zh/whats-new.md:10]:
```suggestion
* [Manifest 窗格中的协议处理程序](/zh/blog/new-in-devtools-105/#manifest)
```

File: site/_includes/partials/devtools/zh/whats-new.md [hunk 23]
```diff
@@ -3,16 +3,17 @@
...
-* [Sourcemap improvements](/zh/blog/new-in-devtools-105/#sourcemaps)
-* [Miscellaneous highlights](/zh/blog/new-in-devtools-105/#misc)
+* [录制面板中的分步重放](/zh/blog/new-in-devtools-105/#recorder)
+* [支持录制面板中的鼠标悬浮事件](/zh/blog/new-in-devtools-105/#recorder-hover)
+* [性能洞察面板中的最大内容绘制 （LCP）](/zh/blog/new-in-devtools-105/#lcp)
+* [将闪烁的文本（FOIT、FOUT）识别为布局变化的潜在根本原因](/zh/blog/new-in-devtools-105/#foit-fout)
+* [Manifest 窗格中的协议处理程序](/zh/blog/new-in-devtools-105/#manifest)
+* [元素面板中的顶层徽章](/zh/blog/new-in-devtools-105/#top-layer)
+* [在运行时附加 Wasm 调试信息](/zh/blog/new-in-devtools-105/#wasm)
+* [调试时支持实时编辑 ](/zh/blog/new-in-devtools-105/#live-edit)
```
Comment (jecfish) [inline site/_includes/partials/devtools/zh/whats-new.md:13]:
```suggestion
* [调试时支持实时编辑](/zh/blog/new-in-devtools-105/#live-edit)
```

File: site/_includes/partials/devtools/zh/whats-new.md [hunk 24]
```diff
@@ -3,16 +3,17 @@
...
+* [支持录制面板中的鼠标悬浮事件](/zh/blog/new-in-devtools-105/#recorder-hover)
+* [性能洞察面板中的最大内容绘制 （LCP）](/zh/blog/new-in-devtools-105/#lcp)
+* [将闪烁的文本（FOIT、FOUT）识别为布局变化的潜在根本原因](/zh/blog/new-in-devtools-105/#foit-fout)
+* [Manifest 窗格中的协议处理程序](/zh/blog/new-in-devtools-105/#manifest)
+* [元素面板中的顶层徽章](/zh/blog/new-in-devtools-105/#top-layer)
+* [在运行时附加 Wasm 调试信息](/zh/blog/new-in-devtools-105/#wasm)
+* [调试时支持实时编辑](/zh/blog/new-in-devtools-105/#live-edit)
+* [在样式窗格的规则中查看和编辑@scope](/zh/blog/new-in-devtools-105/#scope)
+* [Sourcemap 改进](/zh/blog/new-in-devtools-105/#sourcemaps)
+* [其他亮点](/zh/blog/new-in-devtools-105/#misc)
```
Comment (jecfish) [inline site/_includes/partials/devtools/zh/whats-new.md:16]:
```suggestion
* [其他更新](/zh/blog/new-in-devtools-105/#misc)
```

**Comment by aquaMAX** [comment]
@jecfish done the comment merge

**Review by aquaMAX — COMMENTED** [review]
looks good but it seem i can't approve my own PR

**Comment by jecfish** [comment]
Thanks @aquaMAX, author cannot merge own PR. :) I will merge it.

**Review by jecfish — APPROVED** [review]

**Pull request merged**